backend/routes/debug.ts

```diff
@@ -1,5 +1,29 @@
-import express from "express";
+import express, { Request } from "express";
+import { getVideoDetailByIds, getVideosByQuery } from "../rating/youtube";
+import { BadRequestError } from "../utils/errors";
 
 const debug = express.Router();
 
+const VIDEOS_PER_QUERY = 100;
+
+debug.post("/", async (req, res, next) => {
+  const { query } = req.body;
+
+  if (!query) {
+    next(new BadRequestError("query is missing"));
+    return;
+  }
+
+  const videos = await getVideosByQuery(query, VIDEOS_PER_QUERY);
+
+  const ids = [];
+  for (const video of videos.data.items) {
```

Comment:
Probably worth to take a look at Array.Map

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map